standard/region.lua

```diff
@@ -0,0 +1,52 @@
+local region = {}
+local Class = require('Module:Class')
+local Flag = require('Module:Flags')
+local String = require('Module:String')
+local Lua = require('Module:Lua')
+local regionData = mw.loadData('Module:Region/Data')
+local countryToRegionData = Lua.loadDataIfExists('Module:Region/CountryData', {})
+
+function region.run(args)
+	args = args or {}
+	local regionEntry = args.region
+	local onlyRegion = args.onlyRegion == 'true'
+	local onlyDisplay = args.onlyDisplay == 'true'
+
+	if String.isEmpty(regionEntry) then
+		local country = Flag._CountryName(args.country)
+		regionEntry = countryToRegionData[string.lower(country)]
+		if String.isEmpty(regionEntry) then
+			return ''
+		end
+	end
+
+	regionEntry = string.lower(regionEntry)
```

Comment:
This var keeps being re-used, which makes it hard to understand this code

Reply:
renamed it to `region` and added a few comments